### PR TITLE
Remove item when values are empty in map

### PR DESCRIPTION
**Pull request by kerthcet**
/kind cleanup
/assign @elezar

### CODE REVIEW

File: cmd/nvidia-dra-controller/allocations.go [hunk 1]
```diff
@@ -103,6 +103,9 @@ func (p *PerNodeAllocatedClaims) RemoveNode(claimUID, node string) {
 	}
 
 	delete(p.allocations[claimUID], node)
+	if len(p.allocations) == 0 {
```
Comment (elezar) [inline cmd/nvidia-dra-controller/allocations.go:106]: Should this be:
```suggestion
	if len(p.allocations[claimUID]) == 0 {
```
instead?

**Review by elezar — CHANGES_REQUESTED** [review]
Thanks for the submission. One question.

File: cmd/nvidia-dra-controller/allocations.go (hunk 1 above)
Comment (kerthcet) [reply cmd/nvidia-dra-controller/allocations.go]: Ah yes, sorry for the stupid, added tests for validation.

**Comment by elezar** [comment]
@kerthcet I see the linter is complaining due to imports and fmt errors. Once these are addressed it should be good to go.

**Comment by kerthcet** [comment]
Actually I have no idea why this is failed but I changed the import as you did, can you help to approve this workflow, and see what will happen.

**Comment by elezar** [comment]
I have just created #49 so that you can run:
* `make fmt`
* `make goimports`

Locally to address this.

**Review by elezar — APPROVED** [review]

**Comment by elezar** [comment]
Also, please just squash the changes into a single commit.

**Comment by kerthcet** [comment]
Done @elezar

**Pull request merged**